MachineLearning_Engine/Query/Error.cs

```diff
@@ -0,0 +1,52 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.MachineLearning;
+using BH.oM.Reflection;
+using BH.oM.Reflection.Attributes;
+using BH.Engine.MachineLearning;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace BH.Engine.MachineLearning
+{
+    public static partial class Query
+    {
+        /*************************************/
+        /**** Public Fields              ****/
+        /*************************************/
+
+        [Description("Finds the The coefficient of determination R^2 of the given regression model.")]
+        [Input("model", "The linear regressor model used for inference")]
+        [Input("x", "Training data as a list of 2-elements list.")]
+        [Input("y", "Target values as a list of 2-elements list.")]
+        [Output("r2", "The coefficient of determination R^2 of the prediction.")]
+        public static Tensor Error(LinearRegression model, Tensor x, Tensor y)
+        {
+            return new Tensor(BH.Engine.MachineLearning.Compute.Invoke("LinearRegression.score", model, x, y)); ;
```

Comment:
I think the call is to `LinearRegression.error`?
```suggestion
            return new Tensor(BH.Engine.MachineLearning.Compute.Invoke("LinearRegression.error", model, x, y)); ;
```

Reply:
Good catch. My bad.